### DISCUSSION TITLE
Welcome to icepyx Discussions!

### DISCUSSION THREAD

**Discussion by JessicaS11** [icesat2py/icepyx#308] [Announcements]
## 👋 Welcome to icepyx Discussions!
We’re excited to open up the icepyx GitHub Discussions space to continue growing our community. We hope that you:
  * Ask questions about ICESat-2, icepyx, and collaborative development.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.
  * Share your research.

  Feel free to comment below with an introduction of yourself and tell us about what you do with this community. We're excited to get to know you!

**Comment by JessicaS11** [comment]
I'm Jessica - I am the lead maintainer and community manager for icepyx and really enjoy helping people build their confidence and collaborative development skills.

**Reply by kburgos6** [reply]
Hello Jessica! My name is Kiara Burgos and I am working with ICESAT-2 data for a summer research project at NASA Goddard. I am trying to use surface reflectance data to distinguish lava tubes and I am having trouble with the reader function. It is having trouble merging latitude, longitude, and reflectance variables. Do you have any suggestions? I included a screenshot of the error I am getting and the line of code where it's failing 

<img width="789" alt="Screenshot 2023-07-12 at 11 23 47 AM" src="https://github.com/icesat2py/icepyx/assets/116227216/08e7abed-76d4-4d8d-966c-1981fb62ecdc">
<img width="572" alt="Screenshot 2023-07-12 at 11 24 59 AM" src="https://github.com/icesat2py/icepyx/assets/116227216/27619d09-b0cf-4c52-bf65-52391f309000">

**Reply by JessicaS11** [reply]
This discussion post is addressed in #430

**Comment by louis020393** [comment]
Hello Jessica <br>
I am Louis and I want to download ICESAT-2 data. I do not know if it is the good place to ask the question. We are using :
```python
region_a = icepyx.Query('ATL03', spatial_extent , date_range)
region_a.earthdata_login()
region_a.download_granules('data_download')
```
Since the 0.7.0 version of icepyx, the authentification is done with earthacess lib. I do not find on the documentation how to put proxies to the request session (get_session() line 193 in auth.py) ? Is there any post about this subject ?<br>
Best regards :)
